src/pages/feed.js

```diff
@@ -100,36 +100,40 @@ export default class Feed extends Component {
     )
     const image = (index) => (
       <div>
-        <LazyLoadImage
-          className="media"
-          style={{ maxHeight: '50vh', height: 'auto', width: 'auto' }}
-          src={`https://ipfs.io/ipfs/${
-            this.state.items[index].token_info.artifactUri.split('//')[1]
-          }`}
-          alt="💥"
-        />
+        <a href={`/objkt/${this.state.items[index].token_id}`}>
```

Comment:
This should be a Link from react-router-dom as it refers to a route change.

Reply:
Cool, I updated that @andrevenancio